### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by UjjwalBudha**
completed the task

### CODE REVIEW

**Review by milandangol — CHANGES_REQUESTED** [review]

File: ansible/inventory.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[localhost]
```
[the remark this answers is not in the document]
Comment (UjjwalBudha) [reply ansible/inventory.yml]: fixed the issue

File: ansible/roles/website/handlers/main.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+- name: Restart nginx
+  ansible.builtin.service:
+    name: nginx
+    state: restarted
+
+- name: Start nginx
```
[the remark this answers is not in the document]
Comment (UjjwalBudha) [reply ansible/roles/website/handlers/main.yml]: removed unused handler

File: ansible/roles/website/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- name: Remove all files in /usr/share/nginx/html
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: absent
+  with_fileglob:
+    - "{{ website_dir }}"
+
+- name: Clone portfolio repository
+  ansible.builtin.git:
+    repo: 'https://github.com/UjjwalBudha/portfolio.git'
```
[the remark this answers is not in the document]
Comment (UjjwalBudha) [reply ansible/roles/website/tasks/main.yml]: changed the gh repo as a variable

File: ansible/playbook.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+- hosts: 127.0.0.1
```
[the remark this answers is not in the document]
Comment (UjjwalBudha) [reply ansible/playbook.yml]: changed 127.0.0.1 to localhost

**Review by milandangol — APPROVED** [review]

**Pull request merged**